cypress/integration/07-signin.test.js

```diff
@@ -16,6 +16,12 @@ describe('signin', () => {
     cy.get('input[name=email]').should('exist');
   });
 
+  it('trims the email when it has trailing spaces', () => {
+    cy.visit('/signin');
+    cy.get('input[name=email]').type('  user@opencollective.com  ');
+    cy.get('button[type=submit]').should('not.be.disabled');
+  });
```

Comment:
What about test the trim, changing the field and check if has no spaces?

Reply:
done!